CONTRIBUTING.md

```diff
@@ -0,0 +1,121 @@
+# Development Guide: Main Feast Repository
+> The higher level [Development Guide](https://docs.feast.dev/contributing/development-guide)
+> gives guidance on contributing to Feast codebase as a whole.
+
+### Overview
+This guide is targeted at developers looking to contribute to Feast components in
```

Comment:
Not talking about the other repositories is a bit weird. I think as a contributor I'd like to know how to contribute to them all. So basically:
1. Knowing which repos/components exist
2. Knowing what they are for
3. Knowing where to find instructions on contribution.

Doesn't have to be super detailed, but it seems like the current guide is a dead end.